src/server/scripts/Northrend/Ulduar/HallsOfStone/instance_halls_of_stone.cpp

```diff
@@ -40,104 +63,27 @@ class instance_halls_of_stone : public InstanceMapScript
             {
                 SetHeaders(DataHeader);
                 SetBossNumber(EncounterCount);
+                LoadObjectData(creatureData, gameObjectData);
                 LoadDoorData(doorData);
             }
 
-            void OnCreatureCreate(Creature* creature) override
-            {
-                switch (creature->GetEntry())
-                {
-                    case NPC_KRYSTALLUS:
-                        KrystallusGUID = creature->GetGUID();
-                        break;
-                    case NPC_MAIDEN:
-                        MaidenOfGriefGUID = creature->GetGUID();
-                        break;
-                    case NPC_SJONNIR:
-                        SjonnirGUID = creature->GetGUID();
-                        break;
-                    case NPC_MARNAK:
-                        MarnakGUID = creature->GetGUID();
-                        break;
-                    case NPC_KADDRAK:
-                        KaddrakGUID = creature->GetGUID();
-                        break;
-                    case NPC_ABEDNEUM:
-                        AbedneumGUID = creature->GetGUID();
-                        break;
-                    case NPC_BRANN:
-                        BrannGUID = creature->GetGUID();
-                        break;
-                    default:
-                        break;
-                }
-            }
-
             void OnGameObjectCreate(GameObject* go) override
             {
                 InstanceScript::OnGameObjectCreate(go);
 
                 switch (go->GetEntry())
                 {
-                    case GO_ABEDNEUM:
-                        AbedneumGoGUID = go->GetGUID();
-                        break;
-                    case GO_MARNAK:
-                        MarnakGoGUID = go->GetGUID();
-                        break;
-                    case GO_KADDRAK:
-                        KaddrakGoGUID = go->GetGUID();
-                        break;
-                    case GO_TRIBUNAL_CONSOLE:
-                        TribunalConsoleGUID = go->GetGUID();
-                        break;
                     case GO_TRIBUNAL_CHEST:
                     case GO_TRIBUNAL_CHEST_HERO:
-                        TribunalChestGUID = go->GetGUID();
+                        // @todo: this should be a tempsummon or a manually spawned spawngroup to avoid reset exploits
```

Comment:
comment no longer relevant, my future commit adds the chest to spawn group controlled by DATA_TRIBUNAL_OF_AGES